src/spm/services/se_pk11.go

```diff
@@ -21,6 +24,12 @@ import (
 	"github.com/lowRISC/opentitan-provisioning/src/pk11"
 )
 
+var (
+	oidECDSAWithSHA256 = asn1.ObjectIdentifier{1, 2, 840, 10045, 4, 3, 2}
```

Comment:
Might be nice to link the RFC where this comes from in a comment?

Reply:
Done.